src/Ending.hpp

```diff
@@ -1,8 +1,46 @@
 #pragma once
 
+#include "AnmVm.hpp"
+#include "Chain.hpp"
 #include "ZunResult.hpp"
+#include "ZunTimer.hpp"
 
 struct Ending
 {
+    Ending();
     static ZunResult RegisterChain();
-};
+    int DeletedCallback();
+
+    u8 unk_00;
+    u8 unk_01;
+    u8 unk_02;
+    u8 unk_03;
+    ChainElem *chainElem;
+    f32 unk1;
```

Comment:
we're so going to confuse this with unk_01 :joy:. Probably rename this to unk_<offset> or unk_04/5/6?